doc/en/user/source/production/config.rst

```diff
@@ -274,17 +274,32 @@ If you wish to change this behavior you can do so through the following properti
 * ``geoserver.xframe.shouldSetPolicy``: controls whether the X-Frame-Options header should be set at all. Default is true.
 * ``geoserver.xframe.policy``: controls what to set the X-Frame-Options header to. Default is ``SAMEORIGIN``. Valid options are ``DENY``, ``SAMEORIGIN`` and ``ALLOW-FROM [uri]``.
 
+These properties can be set either via Java system property, command line argument (-D), environment
+variable or web.xml init parameter.
```

Comment:
Is this still useful? We now have a page dedicated to this sentence: https://docs.geoserver.org/latest/en/user/configuration/properties/index.html

Reply:
It probably isn't necessary but that sentence exists in the current documentation. It just got moved around and shows up as new in the diff.